src/redux.js

```diff
@@ -342,11 +349,11 @@ export default function rootReducer(state = initialState, action) {
     };
   }
   if (action.type === SET_RESERVE_LOCATION) {
```

Comment:
The changes below look like formatting tweaks? Are my editors settings not in sync with yours?

Reply:
Possibly JsPrettier just wasn't run on a particular commit?  It does seem that the spacing was off by one when committed [here](https://github.com/code-dot-org/ml-playground/commit/8058751c41ae869dbbf4b81d1e31fd6999349fef#diff-7f797aa32260ac4cbc8b43b04d62976c18e503bea6cc86be7448f935896ec3cfR362-R367).